stf/feature_engineering/feature_free_days.py

```diff
@@ -44,27 +46,31 @@ def create_holiday_functions():
                 consist of "Is" + the_name_of_the_holiday_to_be_checked
     """
 
-    # Manully generated csv including all dutch holidays for different regions
-    df_holidays = pd.read_csv(HOLIDAY_CSV_PATH, index_col=None)
-    df_holidays["datum"] = pd.to_datetime(df_holidays.datum).apply(lambda x: x.date())
+    if years is None:
+        years = [2020, 2021]
+
+    holidays_object = holidays.CountryHoliday(country, years=years)
```

Comment:
Would `country_holidays` not be more suitable here. Its just a regular dict with the dates and names of the holidays for a certain country.

I think in general it makes more sense to name things according to which data they contain, not how that data is stored. Also in Python everything is an object of course:)